Cargo.lock

```diff
@@ -383,7 +383,7 @@ checksum = "9a3a5bfb195931eeb336b2a7b4d761daec841b97f947d34394601737a7bba5e4"
 [[package]]
 name = "ijson"
 version = "0.1.3"
-source = "git+https://github.com/RedisJSON/ijson?rev=881b96e7941b8dc335863746ac108f6d9ed0b98a#881b96e7941b8dc335863746ac108f6d9ed0b98a"
+source = "git+https://github.com/RedisJSON/ijson?branch=enable-memory-stats#74ef3e42bb2ecec550511a7f75717d12d59fc717"
```

Comment:
Is this temporary? Isn't the branch going to vanish at some point?

Reply:
yes, just until it gets merged to master. (better than directly to a specific sha though.)

Reply:
redirected to master